### DISCUSSION TITLE
General info

### DISCUSSION THREAD

**Discussion by MatejVukosav** [calimero-network/core#837] [Starknet Hackathon]
**Agenda**:
https://x.com/CalimeroNetwork/status/1846166699101311280

**Hackathon thread:**
https://x.com/CalimeroNetwork/status/1843952063576776952

**Announcement:**
https://x.com/CalimeroNetwork/status/1836053248161067144

**Signup form:**
https://docs.google.com/forms/d/e/1FAIpQLScvoDhnqiK4q2oZmN9hukzorrZyGXj7n51f1yIlWawDOdcoLg/viewform?usp=sf_link

**Comment by MatejVukosav** [comment]
Signups are opened until 31.10

**Comment by nathfavour** [comment]
amazing

**Comment by MatejVukosav** [comment]
Signups are extended https://x.com/calimeronetwork/status/1859638430105239987?s=46
